VS/VehicleSignals.rdf

```diff
@@ -5083,6 +5092,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;VoltageSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Voltage sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.